lib/workers/repository/model/commit-message.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { getName } from '../../../../test/util';
```

Comment:
Any special cause for new folder name? `util->model`

Reply:
I did not recognize which architecture approach now used in repository, that is why I though about starting introduce hexagonal architecture. From this point of view `CommitMessage` looks like domain level model for `repository` module